### PR TITLE
manual updates 20221221 fb

### PR DESCRIPTION
**Pull request by moljac**
### Does this change any of the generated binding API's?

New updates 

- `com.google.android.gms:playservices-ads` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-ads-base` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-ads-lite` - 21.3.0 -> 21.4.0
- `com.google.firebase:firebase-ads` - 21.3.0 -> 21.4.0
- `com.google.firebase:firebase-ads-lite` - 21.3.0 -> 21.4.0
- `com.google.firebase:firebase-inappmessaging` - 20.1.3 -> 20.2.0
- `com.google.firebase:firebase-inappmessaging-display` - 20.1.3 -> 20.2.0
- `com.google.firebase:firebase-inappmessaging-display-ktx` - 20.1.3 -> 20.2.0
- `com.google.firebase:firebase-inappmessaging-ktx` - 20.1.3 -> 20.2.0
- `com.google.firebase:firebase-messaging` - 23.0.8 -> 23.1.1
- `com.google.firebase:firebase-messaging-directboot` - 23.0.8 -> 23.1.1
- `com.google.firebase:firebase-messaging-ktx` - 23.0.8 -> 23.1.1
- `com.google.firebase:firebase-perf` - 20.1.1 -> 20.2.0
- `com.google.firebase:firebase-perf-ktx` - 20.1.1 -> 20.2.0
- `com.google.firebase:firebase-storage` - 20.0.2 -> 20.1.0
- `com.google.firebase:firebase-storage-ktx` - 20.0.2 -> 20.1.0
- `com.google.android.libraries.places:places` - 2.0.0 -> 2.1.0
- `com.google.android.libraries.places:places-compat` - 2.0.0 -> 2.1.0
- `com.github.bumptech.glide:annotations` - 4.13.2 -> 4.14.2
- `com.github.bumptech.glide:disk-lrucache` - 4.13.2 -> 4.14.2
- `com.github.bumptech.glide:gifdecoder` - 4.13.2 -> 4.14.2
- `com.github.bumptech.glide:glide` - 4.13.2 -> 4.14.2
- `com.github.bumptech.glide:recyclerview-integration` - 4.13.2 -> 4.14.2
- `com.google.dagger:dagger` - 2.42 -> 2.43.2
- `com.google.errorprone:error_prone_annotations` - 2.15.0 -> 2.16
- `com.google.flatbuffers:flatbuffers-java` - 2.0.8 -> 22.11.23
- `com.squareup.okhttp3:logging-interceptor` - 4.4.1 -> 4.5.0
- `com.squareup.okhttp3:okhttp` - 4.9.3 -> 4.10.0
- `com.squareup.okhttp3:okhttp-urlconnection` - 4.9.3 -> 4.10.0
- `com.squareup.okio:okio-jvm` -  -> 3.0.0
- `androidx.fragment:fragment` - 1.5.3 -> 1.5.5
- `androidx.media:media` - 1.6.0.1 -> 1.6.0
- `com.google.android.gms:playservices-ads` - 21.3.0 -> 21.4.0
- `com.google.android.gms:playservices-ads-lite` - 21.3.0 -> 21.4.0

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1316,24 +1316,24 @@
...
+        "nugetVersion": "120.2.0",
         "nugetId": "Xamarin.Firebase.Perf",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-perf-ktx",
-        "version": "20.3.0",
-        "nugetVersion": "120.3.0",
+        "version": "20.2.0",
```
Comment (jpobst) [inline config.json:1327]: Version downgrade

File: config.json [hunk 2]
```diff
@@ -1596,8 +1596,8 @@
       {
         "groupId": "com.google.android.libraries.places",
         "artifactId": "places-compat",
-        "version": "2.2.0",
-        "nugetVersion": "2.2.0",
+        "version": "2.1.0",
```
Comment (jpobst) [inline config.json:1599]: Version downgrade

File: config.json [hunk 3]
```diff
@@ -1735,17 +1735,17 @@
       {
         "groupId": "com.google.dagger",
         "artifactId": "dagger",
-        "version": "2.44.2",
-        "nugetVersion": "2.44.2",
+        "version": "2.43.2",
```
Comment (jpobst) [inline config.json:1738]: Version downgrade

File: config.json [hunk 4]
```diff
@@ -1801,26 +1801,26 @@
       {
         "groupId": "com.squareup.okhttp3",
         "artifactId": "logging-interceptor",
-        "version": "4.6.0",
-        "nugetVersion": "4.6.0",
+        "version": "4.5.0",
```
Comment (jpobst) [inline config.json:1804]: Version downgrade

File: config.json [hunk 5]
```diff
@@ -1316,24 +1316,24 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-perf",
-        "version": "20.3.0",
-        "nugetVersion": "120.3.0",
+        "version": "20.2.0",
```
[the remark this answers is not in the document]
Comment (moljac) [reply config.json:1319]: I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2

File: config.json (hunk 1 above)
Comment (moljac) [reply config.json:1327]: I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2

File: config.json (hunk 2 above)
Comment (moljac) [reply config.json:1599]: I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2

File: config.json (hunk 3 above)
Comment (moljac) [reply config.json:1738]: I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2

File: config.json (hunk 4 above)
Comment (moljac) [reply config.json:1804]: I specifically asked to review and merge this PR as 1st (or after last GPS one)

From discord:

> is there any way to commit #730 and #731 first?  the issue is that #729 also contains the changes in the other 2

**Pull request merged**